### ISSUE TITLE
[A11Y] Pages lack describing/unique titles

### ISSUE THREAD

**Issue by erikkroes** [OpenConext/OpenConext-myconext#1063]
## Finding

Several pages use the same non-descriptive title like "eduID"

## Location

The `<head>` of multiple pages.

```<title>eduID</title>```

## Problem

Pages can not be identified or distinguished when they lack a proper title.

## Solution

Add unique titles to pages (in the <head> <title> fields). Not sure howmuch effort that is with current routing solution. Ideally you align this with the text you use to link to the page, and the `<h1>` on the page.

## Resources

- https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html
- https://www.w3.org/WAI/WCAG22/Understanding/consistent-identification.html
